### ISSUE TITLE
New Script idea: Notes - Maintaining county

### ISSUE THREAD

**Issue by jlmartinone** [MN-Script-Team/DHS-PRISM-Scripts#662]
A script that case notes when maintaining county is requested and/or responding to the request for maintaining county. 
Please assign this to me.
I would love any suggestions and feedback. 

 @MN-Script-Team/prism-scriptwriters 

Thanks!

**Comment by theVKC** [comment]
@jlmartinone, can I get an update on where you might be at with this?

Also, @MN-Script-Team/prism-scriptwriters and @MN-Script-Team/prism-supervisors: can I get feedback on this?

**Comment by jlmartinone** [comment]
I have a one little thing to fix and then it should be ready for testing. I will try to get to it this week

**Comment by theVKC** [comment]
Good deal! *Still waiting on feedback, everyone else*

**Comment by jannellstearns** [comment]
Great idea on this as well.  Would this script have the ability to link to the decision tree in SIR for complicated cases?  Then it could  be a one and done stop!

**Comment by theVKC** [comment]
Oooooh, I like that.

**Comment by wrightcountykaren** [comment]
The script is a good idea, I would use it regularly, and I love Jannell's suggested enhancement.

**Comment by jlmartinone** [comment]
@jannellstearns  - is there a specific link you want it to open to?
Thanks!

**Comment by jannellstearns** [comment]
The maintaining county decision tree located in the training materials/job aide page in sir

Sent from my iPhone

On Jan 18, 2017, at 9:01 AM, Jodi Martin <notifications@github.com<mailto:notifications@github.com>> wrote:


@jannellstearns<https://github.com/jannellstearns> - is there a specific link you want it to open to?
Thanks!

**Comment by theVKC** [comment]
@levesswe and @gretchent74 are coordinating testing at their counties. Feedback pending. :)

**Comment by levesswe** [comment]
Tested this script and it works as expected.

**Comment by theVKC** [comment]
Woohoo! There's one!

**Comment by heatherallen** [comment]
I had our maintaining county worker test this script.  We just had a few suggestions and maybe these could be enhancements:
1. To prevent the note from being so long maybe combine the worker name and county on the same line in the CAAD note.  And could we add a spot for phone numbers?

2. The word "reviewing county" was a little confusing so a suggestion to clear this up is to have the layout in the dialog box look like this and possibly look at changing the word reviewing to responding?:

From: (your name)                      Requesting County: (your county)
To: (responding worker)             Responding/Reviewing County: (responding county)

3. Give the worker an option to create a follow-up worklist on CAWT.  Our worker puts a worklist out there that states, "Did so and so from such and such county respond regarding maintaining county?".

Other than these things, the script is functional.

**Comment by theVKC** [comment]
Good suggestions, @heatherallen! I'd say yes, we should probably call that an enhancement, as the bulk of the script is functional right now.

**Comment by theVKC** [comment]
@jlmartinone, the script needs instructions before it can go live. Can you throw instructions together in DHS-SIR using the guidelines on our PRISM scripts page?

**Comment by heatherallen** [comment]
@theVKC would you like me to create another issue for these enhancements?

**Comment by theVKC** [comment]
@heatherallen please dooo!

**Comment by wrightcountykaren** [comment]
My thoughts are listed in red below.

Karen Popken
Wright County Child Support Supervisor
Phone: 763-682-7403
How did I do today? Click here to complete our customer service survey.<https://www.surveymonkey.com/r/WCHHS_CSS>

**Comment by theVKC** [comment]
@wrightcountykaren: email response formatting doesn't carry through, so your "red" isn't visible. :)

I believe @heatherallen will be creating a new issue for this, and if you want to comment there when it goes out, or email her directly, that would be swell. :)

**Comment by wrightcountykaren** [comment]
Oh, ok.  I just sent her a reply.  Were you able to see it, or should I have CC’d you on the message as well?

Karen Popken
Wright County Child Support Supervisor
Phone: 763-682-7403
How did I do today? Click here to complete our customer service survey.<https://www.surveymonkey.com/r/WCHHS_CSS>

**Comment by theVKC** [comment]
When you reply to a GitHub email, we ALL get it and no formatting carries over. It becomes public because GitHub is a public record of our work. :)

If you want to ask a specific person a specific question without everyone seeing it, you forward the email, not reply.

**Comment by gretchent74** [comment]
The script was reviewed by maintaining county worker and is working.   She agreed that having the worker name, county, and optional phone number on one line in the caad note would be helpful.  Also to add an option to add a follow up CAWT note.

**Comment by theVKC** [comment]
Glad it's working! Still waiting on instructions, but @jlmartinone was out last month.
